hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/util/FutureMessage.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.mesos.hdfs.util;
+
+import org.apache.mesos.Protos;
+import org.apache.mesos.SchedulerDriver;
+import org.apache.mesos.hdfs.scheduler.Scheduler;
+
+/**
```

Comment:
I have a significant concern here with creating a separate FutureMethod class that executes one line of code. This also seems to be only executed once and called from one location. Let's not create a separate class unless it's absolutely necessary.